### ISSUE TITLE
[Support] Swap the repo behind x/docxml

### ISSUE THREAD

**Issue by wvbe** [denoland/wanted_modules#94]
Hey there!

TLDR; I am the original author of [wvbe/docxml](https://github.com/wvbe/docxml). Soon I'll be leaving my current employer, and they will be taking over on its development -- but [as a fork](https://github.com/fontoxml/docxml), ie. the origin changes username. I am looking for someone to change which repository may publish to [deno.land/x/docxml](https://deno.land/x/docxml), or guidance on how to do so myself.

I see no way to transfer which repository may push to the webhook for publishing to `deno.land/x/docxml`. Because there are no authentication keys involved I presume that deno.land remembers which GH repository pushed originally, and allows only that repo to push. I could not find other information about this on deno.land, but the following FAQ item makes me think it is reasonable to ask for support from Deno:

> **A module is name-squatting or its just made as a joke, can I have it?**
Name squatting is not allowed on the deno.land/x/. If you feel that a module is not currently usable, has not been legitimately under development for more than 90 days, and you have a concrete proposal to publish a well-maintained module in its place, please contact support.

Since there is not an email that I could find anywhere, I decided to try my luck through this GitHub issue. I believe Fonto meets the criteria of maintaining the module and "legitimately" developing it, whereas my source for it will be updated only on a hobby basis at best.

Thank you for any help or guidance you can give me!
- Wybe

**Comment by wvbe** [comment]
I've discovered `support@deno.land` and sent them an email with the above request. Pending their response I'll update (close?) this ticket :)

**Comment by wvbe** [comment]
This issue was resolved, within minutes, by the excellent @crowlKats. Thank you! Now closing issue.
